src/components/App.js

```diff
@@ -39,21 +40,20 @@ class App extends React.Component {
     }
   }
   render() {
-    if (this.props.currentUser) {
-      return (
-        <div>
-          <LoggedOutView appName={this.props.appName} />
-          {this.props.children}
-        </div>
-      );
-    } else {
-      return (
-        <div>
-          <LoggedInView appName={this.props.appName} />
-          {this.props.children}
-        </div>
-      );
-    }
+    //  if (this.props.currentUser) {
+    return (
+      <div>
+        <LoggedOutView appName={this.props.appName} />
+        {this.props.children}
```

Comment:
Want to be mindful with this pattern as this is reference as inheritance, where usually composition is better

Reply:
this is super insightful thanks!